### DISCUSSION TITLE
Deployement error with Vercel  digital envelope routines::unsupported

### DISCUSSION THREAD

**Discussion by Awazez** [nuxt/nuxt#10830] [Questions (Nuxt 2)]
When i try to deploy with vercel my nuxt app I've got this error. I can't find a solution to it. 

Thanks. 

Error: error:0308010C:digital envelope routines::unsupported
--
16:56:38.312 | at new Hash (node:internal/crypto/hash:71:19)
16:56:38.313 | at Object.createHash (node:crypto:133:10)
16:56:38.313 | at module.exports (/vercel/path0/node_modules/webpack/lib/util/createHash.js:135:53)
16:56:38.313 | at NormalModule._initBuildHash (/vercel/path0/node_modules/webpack/lib/NormalModule.js:417:16)
16:56:38.313 | at handleParseError (/vercel/path0/node_modules/webpack/lib/NormalModule.js:471:10)
16:56:38.313 | at /vercel/path0/node_modules/webpack/lib/NormalModule.js:503:5
16:56:38.313 | at /vercel/path0/node_modules/webpack/lib/NormalModule.js:358:12
16:56:38.314 | at /vercel/path0/node_modules/webpack/node_modules/loader-runner/lib/LoaderRunner.js:373:3
16:56:38.314 | at iterateNormalLoaders (/vercel/path0/node_modules/webpack/node_modules/loader-runner/lib/LoaderRunner.js:214:10)
16:56:38.314 | at Array.<anonymous> (/vercel/path0/node_modules/webpack/node_modules/loader-runner/lib/LoaderRunner.js:205:4)
16:56:38.315 | at Storage.finished (/vercel/path0/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:55:16)
16:56:38.315 | at /vercel/path0/node_modules/enhanced-resolve/lib/CachedInputFileSystem.js:91:9
16:56:38.315 | at /vercel/path0/node_modules/graceful-fs/graceful-fs.js:123:16
16:56:38.316 | at FSReqCallback.readFileAfterClose [as oncomplete] (node:internal/fs/read_file_context:68:3) {

**Comment by nadar** [comment]
Afaik this is related to the latest node versions. You can either use an older node version (16 or maybe 17, not sure) or use this env option somewhere in your dockerfile `ENV NODE_OPTIONS=--openssl-legacy-provider`.

I am not an expert regarding node and nuxt, but i had that same problem in my gitlab pipelines with node 18. hope this helps

**Reply by nadar** [reply]
related https://github.com/nuxt/nuxt.js/discussions/9941
